ros2netperf/setup.py

```diff
@@ -0,0 +1,47 @@
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = 'ros2netperf'
+
+setup(
+    name=package_name,
+    version='0.19.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+    ],
+    install_requires=['ros2cli'],
+    zip_safe=True,
+    author='Ivan Santiago Paunovic',
+    author_email='ivanpauno@ekumenlabs.com',
+    maintainer='Aditya Pande, Audrow Nash, Michael Jeronimo',
+    maintainer_email='aditya.pande@openrobotics.org, audrow@openrobotics.org, michael.jeronimo@openrobotics.org',  # noqa: E501
+    url='https://github.com/ros2/ros2cli/tree/master/ros2topic',
+    download_url='https://github.com/ros2/ros2cli/releases',
```

Comment:
These should point to this repo right?

Reply:
fixed in https://github.com/ros2/netperf/pull/1/commits/257523aa13b6b81126a5a804648b054b19e39939